cmd/root.go

```diff
@@ -59,17 +61,55 @@ const (
 	formatDefault = "default"
 )
 
+const (
+	scorecardLong = "A program that shows security scorecard for an open source software."
+	scorecardUse  = `./scorecard --repo=<repo_url> [--checks=check1,...] [--show-details]
+or ./scorecard --{npm,pypi,rubgems}=<package_name> [--checks=check1,...] [--show-details]`
+	scorecardShort = "Security Scorecards"
+)
+
+func readPolicy(sp *spol.ScorecardPolicy) {
+	if policyFile != "" {
+		data, err := os.ReadFile(policyFile)
+		if err != nil {
+			log.Fatal(err)
+		}
+		err = sp.Read(data)
+		if err != nil {
+			log.Fatal(err)
+		}
+	}
+}
+
+func ensureChecksHavePolicies(sp *spol.ScorecardPolicy, enabledChecks checker.CheckNameToFnMap) {
```

Comment:
Lets ignore `--checks` if `--policy` is passed? Policy file also has `mode: enabled/disabled` which is same as `--checks`.

Reply:
good point. I don't think we want this. Imagine a user has a policy on disk and they only want to run a subset of checks, or even a single: maybe for debugging, maybe to check why a specific check fails, etc. This would force users to create multiple policy files and becomes a usability concern. So I think we should keep the `--checks` to decide which checks to run, and verify we have a policy for it.

Reply:
Will leave it upto you. My 2 cents - it'll make the code more complex and IMO its not worth optimizing for usability (at least for v0).